src/Settings/SettingsScreenNavigator.tsx

```diff
@@ -141,22 +50,22 @@ const SettingsScreenNavigator = () => {
       <Stack.Screen
         name={SETTINGS_ROUTES.MAIN}
         component={SettingsTabNavigator}
-        options={{title: intl.formatMessage(messages.settingsTitle)}}
+        options={{title: strings.settingsTitle}}
       />
       <Stack.Screen
         name={SETTINGS_ROUTES.CHANGE_WALLET_NAME}
         component={ChangeWalletName}
-        options={{title: intl.formatMessage(messages.changeWalletNameTitle)}}
+        options={{title: strings.changeWalletNameTitle}}
       />
       <Stack.Screen
         name={SETTINGS_ROUTES.TERMS_OF_USE}
         component={TermsOfServiceScreen}
-        options={{title: intl.formatMessage(messages.termsOfServiceTitle)}}
+        options={{title: strings.termsOfServiceTitle}}
       />
-      <Stack.Screen
+      <Stack.Screen //
```

Comment:
I think this is a comment that didn't survive.

Reply:
it's only there to prevent formatting to a single line. I'll change it to use `// prettier-ignore` to be clearer